src/amo/components/Addon/index.js

```diff
@@ -107,8 +107,15 @@ export class AddonBase extends React.Component {
     // of an error.
     if (!errorHandler.hasError()) {
       if (addon) {
+        const { slug } = params;
+
+        // make sure slug is number or string without leading -
+        // before redirecting
```

Comment:
We tend to write comments as full English sentences.

Reply:
ahh! ok..